src/app/layouts/organization-layout.component.ts

```diff
@@ -80,4 +80,53 @@ export class OrganizationLayoutComponent implements OnInit, OnDestroy {
         } catch { }
         this.enterpriseTokenPromise = null;
     }
+
+    get showMenuBar() {
+        return this.showManageTab || this.showToolsTab || this.organization.isOwner;
+    }
+
+    get showManageTab(): boolean {
+        return this.organization.canManageUsers ||
+            this.organization.canManageAssignedCollections ||
+            this.organization.canManageAllCollections ||
+            this.organization.canManageGroups ||
+            this.organization.canManagePolicies ||
+            this.organization.canAccessEventLogs;
+    }
+
+    get showToolsTab(): boolean {
+        return this.organization.canAccessImportExport || this.organization.canAccessReports;
+    }
+
+    get showEnterprisePortalButton(): boolean {
+        return this.organization.useBusinessPortal && this.organization.canAccessBusinessPortal;
+    }
+
+    get toolsRoute(): string {
+        return this.organization.canAccessImportExport ?
+            'tools/import' :
+            'tools/exposed-passwords-report';
+    }
+
+    get manageRoute(): string {
```

Comment:
The default view for `manage` is the mange-users screen. Since this is a dedicated permission, it's possible some folks with be able to manage some things but not manage users. There isn't a landing page that would be accessible to anyone with any "mange" permission, so I'm checking permissions and returning whatever the first tab in order of appearance the logged in user has access to.